apps/bettafish/src/app/pages/content-views/news-page/news-page.component.ts

```diff
@@ -26,7 +26,7 @@ export class NewsPageComponent implements OnInit {
     ) {}
 
     ngOnInit(): void {
-        combineLatest(this.viewState.currContent$, this.route.queryParamMap)
+        combineLatest([this.viewState.currContent$, this.route.queryParamMap])
```

Comment:
Handles warning by converting parameters to array; separate parameters in this method is deprecated